### ISSUE TITLE
Non-Functional Testing Addition - Accessibility testing

### ISSUE THREAD

**Issue by rob212** [anas-qa/Quality-Assurance-Road-Map#11]
Great work so far! 
I would suggest that **Accessibility Testing** should be added under Non-Functional Testing section. 

In more detail this includes a knowledge of:

- WCAG (W3C Web Content Accessibility Guidelines) - https://www.w3.org/WAI/standards-guidelines/wcag/
- Screen reader technology such as NVDA or JAWS (Windows), VoiceOver (Mac) 
-  Checklists via the a11y project - https://a11yproject.com/
- Accessibility checker browser plugin tools such as WAVE, Axe, WCAG Color contrast checker or Dark Mode checker
- Automated pipeline tooling such as pa11y - https://pa11y.org/

Happy to assist in adding these and contributing.

**Comment by brunopulis** [comment]
To complement your infos maybe its a good idea link the [Awesome A11y](https://github.com/brunopulis/awesome-a11y).
